web/src/main/webapp/libs/app.bundle.min.js

```diff
@@ -119694,6 +119768,25 @@
         }
         return e
     });
+    T.b.register("features/inprogress-list"), T.c.register("features/inprogress-list", (e = function() { // BAO
+        return []
+    }(), t) => {
+        if (InProgressListEnabled()) switch (t.type) {
+            case "UPDATE_INPROGRESS_LIST":
+                return function(e, t) {
+                        if (t && Array.isArray(t)) {
+                        return t.sort(function(a, b) {
+                            if( a.date > b.date ) return 1;
+                            if( a.date < b.date ) return -1;
+                            return 0;
+                        })
+                    }
```

Comment:
Update the "features/inprogress-list" state with the second argument (t) when the "UPDATE_INPROGRESS_LIST" event is dispatched.
In addition, I sort the list by the start time of the meeting.